addons/debugger/userscript.js

```diff
@@ -0,0 +1,296 @@
+export default async function ({ addon, global, console, msg }) {
+  let workspace, showingConsole;
+
+  const container = document.createElement("div");
+  container.className = "sa-debugger-container";
+  const buttonContainer = document.createElement("div");
+  buttonContainer.className = addon.tab.scratchClass("button_outlined-button", "stage-header_stage-button");
+  const buttonContent = document.createElement("div");
+  buttonContent.className = addon.tab.scratchClass("button_content");
+  const buttonImage = document.createElement("img");
+  buttonImage.className = addon.tab.scratchClass("stage-header_stage-button-icon");
+  buttonImage.draggable = false;
+  buttonImage.src = addon.self.dir + "/debug.svg";
+  buttonContent.appendChild(buttonImage);
+  buttonContainer.appendChild(buttonContent);
+  container.appendChild(buttonContainer);
+  buttonContainer.addEventListener("click", () => toggleConsole(true));
+
+  const vm = addon.tab.traps.vm;
+  addon.tab.addBlock("sa-log %s", ["content"], ({ content }, thread) => {
```

Comment:
Please, do, not, name, this, "sa". Just do a proper conflict handling.

Reply:
What's wrong with naming it sa, and what's "proper conflict handling"?

Reply:
> Just do a proper conflict handling.

Garbo made it so that if projects have a `sa-log` custom block, it won't run at all if this addon is enabled. So if we keep this behavior we'll need to keep the sa prefix.

Reply:
> Garbo made it so that if projects have a sa-log custom block, it won't run at all if this addon is enabled

Prior to this, it would run the custom block with its arguments all set to their default value. This is a more complicated problem to fix than it seems. I'll take another look at some point